security/s2a/internal/crypter/common.go

```diff
@@ -1,9 +1,9 @@
 package crypter
 
 const (
-	// gcmTagSize is the tag size in bytes. From crypto/cipher/gcm.go in the Go
-	// crypto library.
-	gcmTagSize = 16
+	// tagSize is the tag size in bytes. From crypto/cipher/gcm.go and
+	//crypto/chacha20poly1305  in the Go crypto library
```

Comment:
Please remove the library references and simply say `the tag size in bytes for AES-128-GCM-SHA256, AES-256-GCM-SHA384, and CHACHA20-POLY1305-SHA256.`.